druid/src/widget/flex.rs

```diff
@@ -138,6 +140,14 @@ impl<T: Data> Flex<T> {
         };
         self.children.push(child);
     }
+
```

Comment:
biggest nit in the world: I put builder functions right below constructors and before all other methods.

Reply:
Done.